__WTF/Scripts/Weapons/Health.cs

```diff
@@ -25,6 +25,9 @@ public class Health : MonoBehaviour, IHealth
 		[SerializeField, Tooltip("Indicates whether gameObject is destroyed when health is zero"), BoxGroup("Health Settings")]
 		private bool m_destroyOnKilled;
 
+		[SerializeField, Min(0), Tooltip("How long to wait until destroying the gameobject"), BoxGroup("Health Settings")]
+		private float m_destroyDelay = 2f;
```

Comment:
Set default value to 0. That is a more universal case for all games

Reply:
Done 👍